cfgov/teachers_digital_platform/views.py

```diff
@@ -20,6 +20,7 @@
 
 _tdp = 'teachers_digital_platform'
 _signer = signing.Signer()
+_unexpected_error_uri = '../../../assess/survey/'
```

Comment:
Hard-coding URL paths in Django apps is an anti-pattern. Instead, you should be [naming your URLs and using `reverse()` to look them up by name](https://docs.djangoproject.com/en/2.2/topics/http/urls/#reverse-resolution-of-urls).

Reply:
@willbarton The path is a Wagtail page. Any suggestion for how to get a URL to that?

Reply:
I would certainly not recommend hard-coding Wagtail page URLs in code.

Reply:
@willbarton What I'm saying is the only reasonable option for error page is an existing wagtail page, but I don't how I'd use reverse() to get a link to that.

Reply:
Yeah, I think we should have some prior art for discovering the URL of a particular page using its model. I think that's the correct approach — I just need some time to work up some suggested pseudocode.

Reply:
@willbarton Could you find that code? Can we instead (somehow?) hardcode the URL of the wagtail page so it can't be moved AND create a cypress test to ensure the survey links end up on that page?

Reply:
@mrclay I'm sorry, the past few days have been tremendously busy.

As to what you suggest, no, I don't think that would be any better. Late last year we did a massive reorganization of a large amount of content in the CMS, which is what makes me wary about assuming any particular URL structure as opposed to looking up the page we want here.

However, since this requires that that particular page exists, I think the best we can do without converting these views into routable pages and using their Wagtail page tree, is to use its page id to look it up. It looks like the destination page is `4301`, so here's so pseudocode to do that lookup and get the URL:

```
from v1.models import SublandingPage
destination_page = SublandingPage.objects.get(pk=4301)
destination_url = destination_page.get_url(request)
```

Reply:
@willbarton Are you sure the pk for https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/assess/survey/ is 4301?

Reply:
I've been told 15596, going with that...